### ISSUE TITLE
If I've date zoomed, and i click `view underlying data` then I should have date zoom applied

### ISSUE THREAD

**Issue by TuringLovesDeathMetal** [lightdash/lightdash#8055]
### Description

date zoom should be applied (not the default date zoom) if I click on `view underlying data` in a chart within a dashboard

**Comment by rephus** [comment]
I don't think it makes sense to apply zoom here, underlying data is already filtering the time interval, we don't need to change anything here 

See thread: https://lightdash.slack.com/archives/C067071JKKN/p1701700092507989
